src/home/system/SIMOS/Link.json

```
@@ -3,6 +3,11 @@
   "description": "Reference to another entity.",
   "name": "Link",
   "attributes": [
+    {
+      "attributeType": "string",
+      "type": "dmss://system/SIMOS/BlueprintAttribute",
+      "name": "type"
+    },
```

```suggestion
    {
      "attributeType": "string",
      "type": "dmss://system/SIMOS/BlueprintAttribute",
      "name": "type"
    },
```
Is this necessary? Seeing as an identical attribute is defined in the [`system/SIMOS/Blueprint`](https://github.com/equinor/data-modelling-storage-service/blob/feat/validate-on-endpoints/src/home/system/SIMOS/Blueprint.json#L29,L33) blueprint, from which this blueprint inherits?